frontend/src/contexts/connexionContext.jsx

```diff
@@ -0,0 +1,20 @@
+import { createContext, useState } from "react";
+import { auth } from "../firebase-config";
+
+const ConnexionContext = createContext();
+
+export function ConnexionContextProvider({ children }) {
+  const [userInfo, setUserInfo] = useState({
+    auth,
+    info: {},
+  });
+
+  return (
+    // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
Ceci est interdit

Reply:
le linter dit la chose suivante :

The object passed as the value prop to the Context provider (at line 13) changes every render. To fix this consider wrapping it in a useMemo hook.

Or, nous voulons en effet que ce qui est dans le contexte re-render. c'est un contexte de connexion, donc tous les composants doivent savoir si oui ou non un utilisateur s'est connecté ou pas